packages/ComponentLibrary/src/components/Modal/styles.ts

```diff
@@ -0,0 +1,20 @@
+import { CSSProperties } from "react";
+import { NEUTRAL_50, NEUTAL_50_DARK, NEUTRAL_900 } from "../../colors";
+
+const styles: { [key: string]: CSSProperties} = {
+  boxStyles: {
+    position: 'absolute',
+    width: '18.75rem',
```

Comment:
This should be defined by Santi, but I don't think it is correct that these dimensions are hardcoded.